packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -291,4 +293,104 @@ describeNoCompat("Container", (getTestObjectProvider) => {
         assert.strictEqual(container.closed, true);
         assert.strictEqual(pendingLocalState.url, (container.resolvedUrl as IFluidResolvedUrl).url);
     });
+
+    it("can call connect() and disconnect() on Container", async () => {
+        const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+            runtime.IFluidHandleContext.resolveHandle(request);
+        const runtimeFactory = (_?: unknown) => new TestContainerRuntimeFactory(
+            TestDataObjectType,
+            getDataStoreFactory(),
+            {},
+            [innerRequestHandler]);
+        const localTestObjectProvider = new TestObjectProvider(
+            Loader,
+            provider.driver,
+            runtimeFactory);
+
+        const container = await localTestObjectProvider.makeTestContainer() as Container;
+        await timeoutPromise(
+            (resolve) => container.once("connected", () => resolve()),
+            {durationMs: timeoutMs, errorMsg: "container initial connection timeout"},
```

Comment:
The error message already includes the timeout length, so with this change it would look like:
```
Error: container initial connection timeout (500ms)(500ms)
```